src/index.jade

```diff
@@ -44,6 +44,9 @@ html(lang="en")
             li
               a(ui-sref='signup') 
                 | Sign Up
+            li(ng-controller="LogoutController as loutCtrl")
```

Comment:
don't use controllers inline. Add a controller to the top application on the routes